### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**
image
image
image
image

### CODE REVIEW

File: blog/__pycache__/__init__.cpython-311.pyc
Comment (EdAlekseiev) [inline blog/__pycache__/__init__.cpython-311.pyc:1]: Add `__pycache__` to .gitignore and delete these folders from git

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+db.sqlite3
+__pycache__
```
Comment (olena-hul) [inline .gitignore:10]: Add empty line

File: blog/urls.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+
+from blog.views import PostListView, PostDetailView, CommentaryCreateView
+
+
+app_name = "blog"
+
+urlpatterns = [
+    path("", PostListView.as_view(), name="index"),
+    path("posts/<int:pk>", PostDetailView.as_view(), name="post-detail"),
```
Comment (olena-hul) [inline blog/urls.py:10]: Add slash to the url

File: blog_system/asgi.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for blog_system project.
```
Comment (olena-hul) [inline blog_system/asgi.py:2]: Not all pycache folders are deleted

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: blog_system/asgi.py (hunk 3 above)
Comment (EdAlekseiev) [reply blog_system/asgi.py:2]: Still not fixed

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: blog/migrations/__pycache__/__init__.cpython-311.pyc
[the remark this answers is not in the document]
Comment (goldenuni) [reply blog/migrations/__pycache__/__init__.cpython-311.pyc:1]: sorry for my inattentive(

**Review by olena-hul — APPROVED** [review]

**Pull request closed without merging**